warehouse/configuration_testing/configuration_testing.go

```diff
@@ -19,16 +19,38 @@ var (
 	fileManagerTimeout      time.Duration
 )
 
```

Comment:
It is more idiomatic for the filename to be `configuration_test.go` instead of `configuration_testing.go`

Reply:
Unfortunately, it's my bad. This is a package `configuration_test` which doesn't follow the [go package convention](https://go.dev/blog/package-names). Renaming the package name to something more specific.

Reply:
Making this package validations.